Description: Remove use of symbolic icons in Nautilus
 Remove all "symbolic" namespace icons from Nautilus so that the user's
 chosen icon theme is used throughout GNOME.
 .
 nautilus (1:3.4.2-1.2) two; urgency=low
 .
   * Remove all symbolic icons
Author: Ikey Doherty <ikey@solusos.com>

---
Reviewed-By: Ikey Doherty <ikey@solusos.com>
Last-Update: <2012-06-24>

--- nautilus-3.4.2.orig/src/nautilus-location-entry.c
+++ nautilus-3.4.2/src/nautilus-location-entry.c
@@ -385,12 +385,12 @@ nautilus_location_entry_set_secondary_ac
 	case NAUTILUS_LOCATION_ENTRY_ACTION_CLEAR:
 		gtk_entry_set_icon_from_icon_name (GTK_ENTRY (entry), 
 						   GTK_ENTRY_ICON_SECONDARY,
-						   "edit-clear-symbolic");
+						   "edit-clear");
 		break;
 	case NAUTILUS_LOCATION_ENTRY_ACTION_GOTO:
 		gtk_entry_set_icon_from_icon_name (GTK_ENTRY (entry),
 						   GTK_ENTRY_ICON_SECONDARY,
-						   "go-next-symbolic");
+						   "go-next");
 		break;
 	default:
 		g_assert_not_reached ();
--- nautilus-3.4.2.orig/src/nautilus-window-menus.c
+++ nautilus-3.4.2/src/nautilus-window-menus.c
@@ -1080,7 +1080,7 @@ static const GtkToggleActionEntry main_t
   /* tooltip */              N_("Change the visibility of this window's statusbar"),
                              NULL,
   /* is_active */            TRUE },
-  /* name, stock id */     { "Search", "edit-find-symbolic",
+  /* name, stock id */     { "Search", "edit-find",
   /* label, accelerator */   N_("_Search for Files..."), "<control>f",
   /* tooltip */              N_("Search documents and folders by name"),
 			     NULL,
@@ -1149,7 +1149,7 @@ nautilus_window_create_toolbar_action_gr
 					_("Search documents and folders by name"),
 					NULL));
 	gtk_action_group_add_action (action_group, action);
-	gtk_action_set_icon_name (GTK_ACTION (action), "edit-find-symbolic");
+	gtk_action_set_icon_name (GTK_ACTION (action), "edit-find");
 	gtk_action_set_is_important (GTK_ACTION (action), TRUE);
 
 	g_object_unref (action);
--- nautilus-3.4.2.orig/src/nautilus-notebook.c
+++ nautilus-3.4.2/src/nautilus-notebook.c
@@ -366,7 +366,7 @@ build_tab_label (NautilusNotebook *nb, N
 
 	gtk_widget_set_name (close_button, "nautilus-tab-close-button");
 
-	image = gtk_image_new_from_icon_name ("window-close-symbolic", GTK_ICON_SIZE_MENU);
+	image = gtk_image_new_from_icon_name ("window-close", GTK_ICON_SIZE_MENU);
 	gtk_widget_set_tooltip_text (close_button, _("Close tab"));
 	g_signal_connect_object (close_button, "clicked",
 				 G_CALLBACK (close_button_clicked_cb), slot, 0);
--- nautilus-3.4.2.orig/src/nautilus-places-sidebar.c
+++ nautilus-3.4.2/src/nautilus-places-sidebar.c
@@ -222,7 +222,7 @@ get_eject_icon (NautilusPlacesSidebar *s
 
 	icon_theme = gtk_icon_theme_get_default ();
 	icon_size = nautilus_get_icon_size_for_stock_size (GTK_ICON_SIZE_MENU);
-	icon = g_themed_icon_new_with_default_fallbacks ("media-eject-symbolic");
+	icon = g_themed_icon_new_with_default_fallbacks ("media-eject");
 	icon_info = gtk_icon_theme_lookup_by_gicon (icon_theme, icon, icon_size, 0);
 
 	style = gtk_widget_get_style_context (GTK_WIDGET (sidebar));
--- nautilus-3.4.2.orig/src/nautilus-search-bar.c
+++ nautilus-3.4.2/src/nautilus-search-bar.c
@@ -163,7 +163,7 @@ nautilus_search_bar_init (NautilusSearch
 	bar->details->entry = gtk_entry_new ();
 	gtk_entry_set_icon_from_icon_name (GTK_ENTRY (bar->details->entry),
 					   GTK_ENTRY_ICON_SECONDARY,
-					   "edit-find-symbolic");
+					   "edit-find");
 	gtk_container_add (GTK_CONTAINER (align), bar->details->entry);
 
 	g_signal_connect (bar->details->entry, "activate",
--- nautilus-3.4.2.orig/src/nautilus-progress-ui-handler.c
+++ nautilus-3.4.2/src/nautilus-progress-ui-handler.c
@@ -130,7 +130,7 @@ progress_ui_handler_ensure_status_icon (
 		return;
 	}
 
-	icon = g_themed_icon_new_with_default_fallbacks ("system-file-manager-symbolic");
+	icon = g_themed_icon_new_with_default_fallbacks ("system-file-manager");
 	status_icon = gtk_status_icon_new_from_gicon (icon);
 	g_signal_connect (status_icon, "activate",
 			  (GCallback) status_icon_activate_cb,
